app/routes/titiler/integrated_alerts.py

```diff
@@ -0,0 +1,94 @@
+import os
+from typing import Optional, Tuple
+
+from aenum import Enum, extend_enum
+from fastapi import APIRouter, Depends, Query, Response
+from titiler.core.resources.enums import ImageType
+from titiler.core.utils import render_image
+
+from ...crud.sync_db.tile_cache_assets import get_versions
+from ...models.enumerators.tile_caches import TileCacheType
+from ...models.enumerators.titiler import IntegratedAlertConfidence, RenderType
+from .. import DATE_REGEX, optional_implementation_dependency, raster_xyz
+from .algorithms.integrated_alerts import IntegratedAlerts
+from .readers import AlertsReader
+
+DATA_LAKE_BUCKET = os.environ.get("DATA_LAKE_BUCKET")
+
+router = APIRouter()
+
+dataset = "gfw_integrated_alerts"
+
+
+class GfwIntegratdAlertsVersions(str, Enum):
+    """GFW Integrated Alerts versions.
+
+    When using `latest` call will be redirected (307) to version tagged
+    as latest.
+    """
+
+    latest = "latest"
+
+
+_versions = get_versions(dataset, TileCacheType.cog)
+for _version in _versions:
+    extend_enum(GfwIntegratdAlertsVersions, _version, _version)
+# TODO: add version validation
+
+
+# will turn this on when we're ready to replace tile cache service
+# @router.get(
+#     f"/{dataset}/{{version}}/dynamic/{{z}}/{{x}}/{{y}}.png",
+#     response_class=Response,
+#     tags=["Raster Tiles"],
+#     response_description="PNG Raster Tile",
+# )
+@router.get(
```

Comment:
the `/titiler` endpoint for integrated alerts here is temporary to deploy this and test it without replacing the existing dynamic implementation. there is an item in the roadmap to fully implement this later.